### ISSUE TITLE
semver.satisfies no longer accepts undefined version

### ISSUE THREAD

**Issue by qtomasicchio** [npm/node-semver#278]
Hello,

There seems to be a regression in the last version (6.1.0) on the `satisfies` method.

This was working in version 6.0.0: 
```
semver.satisfies(undefined, '>=1.0.0'); // always return false
```

This no longer the case in 6.1.0:
```
semver.satisfies(undefined, '>=1.0.0'); // throw TypeError: Invalid Version: null
```

To my understanding, this is due to the suppression of this check in https://github.com/npm/node-semver/commit/7b072bd1e5446e60559898dfd45cc8fb4361ef0d
```diff
@@ -1250,10 +1256,6 @@ function hyphenReplace ($0,

// if ANY of the sets match ALL of its comparators, then pass
Range.prototype.test = function (version) {
-  if (!version) {
-    return false
-  }

  if (typeof version === 'string') {
    version = new SemVer(version, this.options)
  }
```

Is this change wanted? Can I submit a PR to fix it ?

Thanks,
Quentin Tomasicchio

**Comment by mohamedbenjelloun** [comment]
I agree, a bit disturbing

**Comment by lilneo** [comment]
Seems relevant

**Comment by isaacs** [comment]
Fixed on 6.1.1.

**Comment by qtomasicchio** [comment]
It works, thanks a lot! :+1:
